extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DefaultTenantConfigResolver.java

```diff
@@ -166,7 +180,11 @@ private TenantConfigContext getStaticTenantContext(RoutingContext context) {
     }
 
     private String resolveStaticTenantId(RoutingContext context) {
-        String tenantId;
+        String tenantId = context.get(OidcUtils.TENANT_ID_ATTRIBUTE);
+        if (isTenantSetByAnnotation(context, tenantId)) {
```

Comment:
@sberyozkin you changed the order of resolvers. Annotation-based should be behind `tenantResolver` according to the https://quarkus.io/version/main/guides/security-openid-connect-multitenancy#static-tenant-resolution. I remember this change was discussed, but effectively documented order is different now?

I am asking because I did some tweaks to this code and need to know if this order should be kept now. I think it's fine, we just should change docs. Your call.

Reply:
@michalvavrik Sure, somewhere, as part of discussing this issue, I suggested the annotation based resolution check should be made before asking the custom resolvers to do the work, I'll tweak the docs

Reply:
Thank you, I wasn't sure.

Reply:
As otherwise it would defeat the purpose of using those annotations :-)